File: Object_Orianted_Programming_For_YTM/C#/AbstractClassExample/AbstractClassExample/Daire.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace AbstractClassExample
{
    public class Daire : Sekil
    {
        public double YariCap { get; set; }

        public Daire(double yariCap)
        {
            YariCap = yariCap;
        }

        public override double AlanHesapla()
        {
            return Math.PI * YariCap * YariCap;
        }
    }
}

File: LabStudy/Pano/src/Daire.java
public class Daire implements IAlan,ICevre {
    private double yaricap;

    public Daire(double yaricap) {
        this.yaricap = yaricap;
    }
    @Override
    public double alanHesapla() {
        return Math.PI * yaricap * yaricap;
    }
    @Override
    public double cevreHesapla() {
        return 2 * Math.PI * yaricap;
    }
}